actions/dashboard/AI/ShowCaseActions.tsx

```diff
@@ -148,11 +149,14 @@ export async function continueShowCaseChatConversation(
                             time={dayjs().format('dddd, MMMM D, YYYY h:mm A')}
                             isUser={false}
                         >
-                            <SuccessMessage
-                                status="success"
-                                message="Assignment marked as completed."
-                                fire={true}
-                            />
+                            <div className="w-full flex flex-col gap-8">
+                                <SuccessMessage
+                                    status="success"
+                                    message="Assignment marked as completed."
+                                    fire={true}
```

Comment:
If prop is true, no need for `fire={true}`. With just `fire` should work

Reply:
jejej true XD